From 43194c25943acd2588aff56cce80485dd31238d6 Mon Sep 17 00:00:00 2001
From: Sumit Gupta <sumitg@nvidia.com>
Date: Mon, 9 Oct 2023 13:54:23 +0530
Subject: [PATCH 5/9] cpufreq: tegra194: remove redundant AND with
 cpu_online_mask

Remove redundant 'AND' with cpu_online_mask as the policy->cpus always
contains only the currently online CPUs.

Upstream-Status: Pending
Suggested-by: Viresh Kumar <viresh.kumar@linaro.org>
Link: https://lore.kernel.org/lkml/20231003050019.a6mcchw2o2z2wkrh@vireshk-i7/
Signed-off-by: Sumit Gupta <sumitg@nvidia.com>
[ Viresh: Fix rebase conflict ]
Signed-off-by: Viresh Kumar <viresh.kumar@linaro.org>
---
 drivers/cpufreq/tegra194-cpufreq.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/cpufreq/tegra194-cpufreq.c b/drivers/cpufreq/tegra194-cpufreq.c
index 9dae6195e0e7..59865ea455a8 100644
--- a/drivers/cpufreq/tegra194-cpufreq.c
+++ b/drivers/cpufreq/tegra194-cpufreq.c
@@ -135,7 +135,7 @@ static void tegra234_set_cpu_ndiv(struct cpufreq_policy *policy, u64 ndiv)
 	struct tegra194_cpufreq_data *data = cpufreq_get_driver_data();
 	u32 cpu;
 
-	for_each_cpu_and(cpu, policy->cpus, cpu_online_mask)
+	for_each_cpu(cpu, policy->cpus)
 		writel(ndiv, data->cpu_data[cpu].freq_core_reg);
 }
 
-- 
2.25.1

